google-cloud-firestore/src/test/java/com/google/cloud/firestore/it/ITAggregateQueryTest.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.firestore.it;
+
+import static com.google.common.truth.Truth.assertThat;
+
+import com.google.cloud.firestore.AggregateQuery;
+import com.google.cloud.firestore.Firestore;
+import com.google.cloud.firestore.FirestoreOptions;
+import com.google.cloud.firestore.Query;
+import com.google.common.base.Preconditions;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+@RunWith(JUnit4.class)
+public class ITAggregateQueryTest {
+
+  private Firestore firestore;
+
+  @Before
+  public void setUpFirestore() {
+    // TODO(dconeybe) Remove the hardcoded "host" and "projectId" once COUNT is supported by prod.
```

Comment:
It appears that the CI doesn't run the integration tests at all. I've opened b/244164955 to look into this, since it would be ideal that they are, indeed, run in CI.

Reply:
Thanks